packages/fxa-auth-server/lib/senders/emails/templates/postAddTwoStepAuthentication/en-US.ftl

```diff
@@ -0,0 +1,6 @@
+postAddTwoStepAuthentication-subject = Two-step authentication enabled
```

Comment:
Worth noting that in the current template the subject is "Two-step verification enabled", and I think is the only place we call it verification not authentication. So I made it consistent.

Reply:
Seems right, but mind making a note of that in the "Confirmation needed" doc?